app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb

```diff
@@ -126,7 +126,7 @@ def connect(_args = {})
     self.class.raw_connect(region, endpoint, access_key, secret_key)
   end
 
-  def verify_credentials(_args = {})
+  def verify_credentials(_args = {}, _options = {})
```

Comment:
The args for this should be `def verify_credentials(_auth_type = nil, _options = {})` (see https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/authentication_mixin.rb#L468)

Reply:
Corrected, thanks!